recipe/meta.yaml

```diff
@@ -27,14 +27,17 @@ requirements:
         - zlib
 
 test:
+    requires:
+        - python {{ environ['PY_VER'] + '*' }}  # [win]
```

Comment:
I think this is the best we have got for now. We can revisit after the VC meeting.